test/helpers/fixtures.coffee

```diff
@@ -73,6 +73,14 @@ module.exports = ->
     'partner_logo_url': 'http://gemini.herokuapp.com/123/miaart-banner.jpg'
     'thumbnail_url': 'http://gemini.herokuapp.com/123/miaart-banner.jpg'
     'featured_article_ids': [ '5522d03ae8e369060053d953' ]
+  fixtures.sections =
```

Comment:
Yeah def, I'll remove it at the "Delete Vertical" step since the `/vertical` endpoint still exists and uses the fixture